sdk/storage/Azure.Storage.DataMovement/tests/Shared/StartTransferDownloadTestBase.cs

```diff
@@ -0,0 +1,810 @@
+        {
+            // Arrange
+            await using IDisposingContainer<TContainerClient> testContainer = await GetDisposingContainerAsync();
+
+            // No Option Download bag or manager options bag, plain download
+            await DownloadObjectsAndVerify(
+                testContainer.Container,
+                waitTimeInSec: 10,
+                size: Constants.KB,
+                objectCount: 1).ConfigureAwait(false);
+        }
+
+        [RecordedTest]
+        public async Task RemoteObjectToLocal_Overwrite_Exists()
+        {
+            // Arrange
+            // Create source local file for checking, and source object
+            await using IDisposingContainer<TContainerClient> testContainer = await GetDisposingContainerAsync();
+            string objectName = GetNewObjectName();
+            string localSourceFile = Path.GetTempFileName();
+            int size = Constants.KB;
+            TObjectClient sourceClient = await GetObjectClientAsync(
+                container: testContainer.Container,
+                objectLength: size,
+                objectName: objectName,
+                createResource: true);
+
+            // Create destination to overwrite
+            string destFile = Path.GetTempFileName();
+
+            // Act
+            // Create options bag to overwrite any existing destination.
+            DataTransferOptions options = new DataTransferOptions()
+            {
+                CreationPreference = StorageResourceCreationPreference.OverwriteIfExists,
+            };
+            List<DataTransferOptions> optionsList = new List<DataTransferOptions> { options };
+            await DownloadObjectsAndVerify(
+                testContainer.Container,
+                waitTimeInSec: 10,
+                size: Constants.KB,
+                objectCount: 1,
+                options: optionsList).ConfigureAwait(false);
+        }
+
+        [RecordedTest]
+        public async Task RemoteObjectToLocal_Overwrite_NotExists()
+        {
+            // Arrange
+            // Create source local file for checking, and source object
+            await using IDisposingContainer<TContainerClient> testContainer = await GetDisposingContainerAsync();
+
+            // Act
+            // Create options bag to overwrite any existing destination.
+            DataTransferOptions options = new DataTransferOptions()
+            {
+                CreationPreference = StorageResourceCreationPreference.OverwriteIfExists,
+            };
+            List<DataTransferOptions> optionsList = new List<DataTransferOptions> { options };
+            await DownloadObjectsAndVerify(
+                testContainer.Container,
+                waitTimeInSec: 10,
+                size: Constants.KB,
+                objectCount: 1,
+                options: optionsList).ConfigureAwait(false);
+        }
+
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/33086")]
```

Comment:
This issue is marked as Closed so is it still happening now?

Reply:
I can check that in a different PR. I know that @jaschrep-msft did some work to make these tests enabled again.